Makefile

```diff
@@ -66,6 +76,7 @@ kind-only: # @HELP deploy the image without rebuilding first
 kind-only:
 	@if [ "`kind get clusters`" = '' ]; then echo "no kind cluster found" && exit 1; fi
 	kind load docker-image --name ${KIND_CLUSTER_NAME} ${DOCKER_REPOSITORY}onos-config:${ONOS_CONFIG_VERSION}
+	kind load docker-image --name ${KIND_CLUSTER_NAME} ${DOCKER_REPOSITORY}config-model-registry:${ONOS_CONFIG_VERSION}
 
 all: build images
 
```

Comment:
You'll have to add the new image to the publish target:

./../build-tools/publish-version ${VERSION} onosproject/config-model-registry